vr4300/cp0.c

```diff
@@ -167,6 +170,9 @@ int VR4300_MFC0(struct vr4300 *vr4300,
   else if (vr4300_cp0_reg_masks[src] == 0x0000000000000BADULL)
     exdc_latch->result = (int32_t) vr4300->regs[VR4300_REGISTER_CP0_0 + 7];
 
+  else if (src + VR4300_REGISTER_CP0_0 == VR4300_CP0_REGISTER_PRID)
+    exdc_latch->result = 0xb22; // Hardcoded revision of a gray NTSC N64
```

Comment:
I don't really believe the color (early 0xB10 consoles would be gray too) or NTSC (got same results on my PAL console) matters here, this comment may cause more confusion than it solves.